drake/systems/onlinePlanning/TrajectoryLibraries/TrajectoryLibrary.m

```diff
@@ -0,0 +1,57 @@
+classdef TrajectoryLibrary    
```

Comment:
i guess the old version was actually in dev: https://github.com/RobotLocomotion/drake/tree/master/drake/systems/trajectories/dev .  happy to have this replace that one.